autoblocks/_impl/testing/run.py

```diff
@@ -155,6 +156,8 @@ async def run_test_case_unsafe(
     Its caller will catch and handle all exceptions.
     """
     async with test_case_semaphore_registry[test_id]:
+        start_time = time.perf_counter()
```

Comment:
note this is happening after acquiring the semaphore lock so won't include time waiting for that